Use telemetryLevel to disable crash reporting (#5970)
microsoft/vscode-docs: docs/getstarted/telemetry.md

**Error Telemetry** - Error telemetry collects information about errors that do not crash the application but are unexpected.

> **Usage Data** - Usage data collects information about how features are used and perform in VS Code, which helps us prioritize future product improvements.

Author: ```suggestion
**Usage Data** - Usage data collects information about how features are used and perform in VS Code, which helps us prioritize future product improvements.
```how about not

> **Usage Data** - Usage data collects information about how features are used and perform in VS Code, which helps us prioritize future product improvements.

Author: ```suggestion
**Usage Data** - Usage data collects information about how features are used and perform in VS Code, which helps the team prioritize future product improvements.
```

Author: This look better

> ## Disable telemetry reporting

Author: ```suggestion
## Disable telemetry reporting
``````suggestion
## Disable telemetry reporting
``````suggestion
## Disable telemetry reporting blah
```

Author: ```suggestion
## Disable telemetry reporting
``````suggestion
## Disable telemetry reporting
``````suggestion
## Disable telemetry reporting blah
```
